lib/QueryValidationVisitor.js

```diff
@@ -0,0 +1,229 @@
+const { getVariableValues } = require('graphql/execution/values.js')
+const {
+  GraphQLString,
+  Kind,
+  getNamedType,
+  isInputObjectType,
+  isListType,
+  isNonNullType,
+  BREAK,
+  valueFromAST,
+  typeFromAST,
+  visit,
+  getDirectiveValues
+} = require('graphql')
+const ValidationError = require('./error')
+const { getConstraintValidateFn, getScalarType } = require('./typeutils')
+const { constraintDirectiveTypeDefsObj } = require('./typeDefs')
+
+// const DEBUG = true
+// const DEBUG2 = true
+
+// function logDebug (message, msg2) {
+//   if (DEBUG) console.log('\n' + message + (DEBUG2 && msg2 ? msg2 : ''))
+// }
```

Comment:
Remove commented out code please :)

Reply:
sure, that was leftovers usefull for development :-)